client/src/main/java/org/spine3/time/ZoneOffsets.java

```diff
@@ -21,29 +21,53 @@
 package org.spine3.time;
 
 import org.spine3.protobuf.Durations;
-import org.spine3.protobuf.Timestamps;
+
+import java.util.Calendar;
 
 import static com.google.common.base.Preconditions.checkArgument;
 
 /**
  * Utilities for working with ZoneOffset objects.
  *
+ * @author Alexander Yevsyukov
+ * @author Alexander Aleksandrov
  * @see ZoneOffset
  */
 public class ZoneOffsets {
 
-    private ZoneOffsets() {}
-
-    public static final ZoneOffset UTC = ZoneOffset.newBuilder().setId("UTC").setAmountSeconds(0).build();
+    private ZoneOffsets() {
+    }
 
+    public static final ZoneOffset UTC = ZoneOffset.newBuilder()
+                                                   .setId("UTC")
+                                                   .setAmountSeconds(0)
+                                                   .build();
 
+    /**
+     * Obtains the ZoneOffset instance using an offset in hours.
+     */
     public static ZoneOffset ofHours(int hours) {
-        checkArgument(Math.abs(hours) < Timestamps.HOURS_PER_DAY, "offset size must be < 24 hours");
+        checkArgument(Math.abs(hours) <= 18, "offset size must be between -18 and 18 hours inclusive");
```

Comment:
Is it symmetrical (18 on both ends)? 

If not have two checks with corresponding messages. And do have constants.